cpp/augment_bible_references.cc

```diff
@@ -457,62 +440,51 @@ void AugmentBibleRefs(const bool verbose, MarcReader * const marc_reader, MarcWr
                 }
 
                 // Put the data into the $a subfield:
-                record.insertSubfield(BibleUtil::BIB_REF_RANGE_TAG, 'a', range_string);
+                bible_reference_tag_field->insertOrReplaceSubfield('a', range_string);
```

Comment:
We have established that bible_reference_tag_field is record.end()) on line 429!  You need to use Record::insertField().